### ISSUE TITLE
Agreements on translation details

### ISSUE THREAD

**Issue by matborowiak** [gatsbyjs/gatsby-pl#12]
### Hej, myśle że przyda się nam taki wątek, żebyśmy mogli przedyskutować pewne szczegóły, poza poszczególnymi PR, bo nikt raczej nie będzie zaglądał w te które zostały już zmergowane.

Zacząłem dopisywać na swoim branch'u terminy do słowniczka. Co robimy z tłumaczeniami typów danych? Array, String, Integer, Object, Null, Undefined itd. ?

Tłumaczymy np. "Array" na "Tablica", czy zostawiamy angielską pisownię ?

**Comment by maciekgrzybek** [comment]
Good idea :) Z jednej strony dobrze by było tłumaczyć na Polski, ale z doświadczenia wiem, że mało kto używa Polskich wersji tych słów. Byłbym za zostawieniem Angielskich wersji.
